README.md

```diff
@@ -0,0 +1,130 @@
+Introduction
+------------
+
+The Vote Up/Down module allows the user to cast votes on some entities in
+a Drupal site with different widgets. This module uses Voting API to implement
+votes and Chaos Tools to provide AJAX support and make custom widgets.
+
+ * For a full description of the module, visit the project page:
+   https://drupal.org/project/vote_up_down
```

Comment:
Let's use current host of d.o, i.e. https://www.drupal.org. Please do this on all d.o URLs, not only this instance.